src/WaterPokemon.java

```diff
@@ -0,0 +1,54 @@
+public class WaterPokemon extends pokemonsuper {
+    private int waterHeight;
+    private boolean alive;
```

Comment:
Deze variabele komt in elke subklasse terug. Had dus ook beter in de superklasse gemogen en hier iets anders